src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim())
+    .map((str) => {
+      return str
+        .trim()
+        .split(':')
+        .map((s) => s.trim());
+    });
+
+  return Object.fromEntries(cssProperties);
```

Comment:
While the chaining of methods is done correctly, the indentation for each chained method should be consistent to improve readability. Align all the methods under the first one.